### DISCUSSION TITLE
After Section Test for NFT rewards

### DISCUSSION THREAD

**Discussion by anihdev** [Cyfrin/foundry-full-course-cu#2015] [General]
Hi!
I'm finding it difficult to engage in the after section tests for NFT and other rewards, can I get direction from anyone?
@PatrickAlphaC

**Comment by EngrPips** [comment]
Hello @anihdev, That is why it is called a challenge. It is to challenge you, and it might take time before you can solve them, so take as much time as you need and do as much research as possible that will make you strong enough for the challenge. I would suggest you look at the contract of the particular challenge on etherscan to see if anyone has successfully solved that challenge and how they did it.

**Reply by anihdev** [reply]
Thanks for sending me a response, I understand what u meant, I’m not really asking for a help to solve the challenge, I’m yet to find the section challenge, when I click the sepolia link it redirects to something like etherscan site. So where can I find the challenge to solve it

**Reply by EngrPips** [reply]
Check this discussion I explained there https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018#discussion-6933519. If anything is confusing, please ask questions

**Reply by anihdev** [reply]
Thanks very, that’s what I’m looking for. It was helpful

**Reply by EngrPips** [reply]
You are welcome. Have fun!

**Comment by anihdev** [comment]
Goodday
I hv a little issue, please I’m still new don’t mind my possibly dumb
questions, I’ve gotten the address of the challenge, how do I look it up on
remix as u said, where should I paste it or how do I go about it?

**Reply by EngrPips** [reply]
Hello, @anihdev. I edited that post and corrected myself. I meant etherscan
